core/tests/test_utils.py

```diff
@@ -43,7 +43,9 @@
 from google.appengine.api import apiproxy_stub_map
 from google.appengine.api import mail
 
+# Model for Oppia explorations for valid model names.
```

Comment:
You don't need to comment these two lines. They're just (fancier) imports.

Reply:
done

Reply:
done